content/blog/_posts/ali-siddiqui-bmc-interview.md

```diff
@@ -0,0 +1,29 @@
+# Moving the needle on inclusive language, with Ali Siddiqui from BMC
+
+Welcome to our series Moving the needle on inclusive language. In this series, we will be talking to leaders in the cloud-native space as they share the inclusive naming initiatives they care about. This will include tools, activities, and results they have discovered along the way.
```

Comment:
```suggestion
Welcome to our series Moving the Needle on inclusive language. In this series, we will be talking to leaders in the cloud native space as they share the inclusive naming initiatives they care about. This will include tools, activities, and results they have discovered along the way.
```